src/App.tsx

```diff
@@ -76,18 +61,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && !loading && !error && posts.length === 0 && (
                   <div className="notification is-warning" data-cy="NoPostsYet">
                     No posts yet
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length > 0 && (
-                  <PostsList
-                    posts={posts}
-                    selectedPostId={selectedPost?.id}
-                    onPostSelected={setSelectedPost}
-                  />
+                {author && !loading && !error && posts.length > 0 && (
```

Comment:
```suggestion
                {author && !loading && !error && !!posts.length && (
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**